### ISSUE TITLE
skin.ini files did not load correctly

### ISSUE THREAD

**Issue by takase1121** [itdelatrisu/opsu#396]
```Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'MenuGlow: 255, 255, 255' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 255"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'SongSelectActiveText: 255, 255, 255' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 255"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'SongSelectInactiveText: 200, 200, 200' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 200"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'SliderBall: 0, 0, 0' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 0"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'SliderTrackOverride: 0, 0, 0 //25,30,33' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 0"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'SpinnerBackground: 200, 200, 200' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 200"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'Combo1: 62, 153, 148' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 153"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'InputOverlayText: 160, 67, 46' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 67"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
Sun Jun 03 15:51:42 SGT 2018 WARN:Failed to read color 'StarBreakAdditive: 229, 95, 65' for file 'C:\osu!\Skins\Nekopara!\skin.ini'.
java.lang.NumberFormatException: For input string: " 95"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at itdelatrisu.opsu.skins.SkinLoader.loadSkin(SkinLoader.java:180)
	at itdelatrisu.opsu.options.Options.loadSkin(Options.java:1498)
	at itdelatrisu.opsu.Utils.init(Utils.java:150)
	at itdelatrisu.opsu.states.Splash.init(Splash.java:105)
	at org.newdawn.slick.state.StateBasedGame.init(StateBasedGame.java:171)
	at org.newdawn.slick.AppGameContainer.setup(AppGameContainer.java:393)
	at itdelatrisu.opsu.Container.runSetup(Container.java:112)
	at itdelatrisu.opsu.Container.start(Container.java:77)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:213)
```

This is the skin.ini I tried to load. I renamed it into skin.txt since .ini extension is not supported.
[skin.txt](https://github.com/itdelatrisu/opsu/files/2065772/skin.txt)

**Comment by Lemmmy** [comment]
That's weird, as they are being trimmed:

https://github.com/itdelatrisu/opsu/blob/66b68a340c69416c610ea3b420594661f6d46514/src/itdelatrisu/opsu/skins/SkinLoader.java#L177-L182

I thought it might be some unicode silliness, so I checked the `skin.ini` and the chars are proper ASCII (`0x39` for the commas, `0x20` for the spaces).

**Comment by itdelatrisu** [comment]
Ah, it's because 6a6182d isn't in the last release. Might do another soon, I guess...

**Comment by itdelatrisu** [comment]
Fixed in the 0.16.1 release.
